source/NVDAObjects/IAccessible/ia2TextMozilla.py

```diff
@@ -684,9 +684,16 @@ def _findNextContent(self, origin, moveBack=False, limitToInline=False):
 				pass
 		return ti, obj
 
-	def move(self, unit, direction, endPoint=None):
+	def move(
+		self,
+		unit: int,
+		direction: str,
```

Comment:
I think you mixed these up.

```suggestion
		unit: str,
		direction: int,
```